pyproject.toml

```diff
@@ -0,0 +1,50 @@
+[tool]
```

Comment:
Another silly question and probably one I should know the answer to -- Pypi official support `toml` now?

Reply:
Yes indeed. To deploy to pypi all you need to do is, `poetry deploy`. It's kinda awesome.

Reply:
I saw that in their documentation -- this looks like it's gonna be really rad. I'm gonna port pyaestro to using it.

Reply:
I've been using for my django plugins for work, it's really easy to get it to work with gitlab and pypi.